packages/rrweb/src/replay/index.ts

```diff
@@ -1085,10 +1131,17 @@ export class Replayer {
       }
       case IncrementalSource.Drag:
       case IncrementalSource.TouchMove:
-      case IncrementalSource.MouseMove:
+      case IncrementalSource.MouseMove: {
+        const pointerId =
+          'pointerId' in d && d.pointerId !== undefined
+            ? d.pointerId
+            : Object.keys(this.pointers).length;
```

Comment:
I can't find my previous comment about this, but this assumption is not good for a few reasons. (Ignore the current state of SDKs where pointerId will always exist in mobile SDKs and never exist on web SDKs)

* Can cause conflicts if a pointerId-less is active while a new pointer (w/ id) is created (or stopped). The reference to this pointer depends on the current number of pointers so can be unstable as `this.pointers` changes
* I forgot my other point :(

Instead, let's make this `-1` since `this.pointers.length` can never be -1, and this lets us in the future mix pointer with and without `pointerId`.